framework-plugins/lisk-framework-chain-connector-plugin/src/schemas.ts

```diff
@@ -35,18 +35,34 @@ export const configSchema = {
 		},
 		encryptedPrivateKey: {
 			type: 'string',
+			description: 'Encrypted privateKey of the relayer',
 		},
 		ccuFee: {
 			type: 'string',
 			format: 'uint64',
+			description: 'Fee to be paid for each CCU transaction',
 		},
 		password: {
 			type: 'string',
+			description: 'Password to decrypt encryptedPrivateKey',
+		},
+		isSaveCCU: {
```

Comment:
how about `storageMode`, `sendMode`? `isSaveCCU` is not grammatically correct

Reply:
the other options are also not covering the context very well, we have 2 options here, save CCU or send CCU  and relating to that it still serves fine, isSaveCCU suggested by @Phanco , but i'm fine if we have any other suggestion